### PR TITLE
Write unit test for offerSearchToolbar component

### PR DESCRIPTION
**Pull request by Mav-Ivan**
image

### CODE REVIEW

File: tests/unit/containers/find-offer/offer-search-toolbar/offerSearchToolbar.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      <OfferSearchToolbar
+        filterActions={filterActions}
+        filters={filters}
+        resetPage={resetPage}
+      />
+    )
+  })
+
+  it('should find right category value', () => {
+    const testValue = 'Music'
```
Comment (tsivkadi) [inline tests/unit/containers/find-offer/offer-search-toolbar/offerSearchToolbar.spec.jsx:29]: move variables to the top

Comment (Mav-Ivan) [reply tests/unit/containers/find-offer/offer-search-toolbar/offerSearchToolbar.spec.jsx]: done

File: tests/unit/containers/find-offer/offer-search-toolbar/offerSearchToolbar.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import React from 'react'
```
Comment (Yurenko) [inline tests/unit/containers/find-offer/offer-search-toolbar/offerSearchToolbar.spec.jsx:1]: Delete it redundant

Comment (Mav-Ivan) [reply tests/unit/containers/find-offer/offer-search-toolbar/offerSearchToolbar.spec.jsx:1]: ok , thanks

**Review by tsivkadi — APPROVED** [review]

File: tests/unit/containers/find-offer/offer-search-toolbar/offerSearchToolbar.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import { render, screen, fireEvent } from '@testing-library/react'
+import { vi } from 'vitest'
+import OfferSearchToolbar from '~/containers/find-offer/offer-search-toolbar/OfferSearchToolbar'
```
Comment (Tolik170) [inline tests/unit/containers/find-offer/offer-search-toolbar/offerSearchToolbar.spec.jsx:3]: organize imports

**Review by MickhaelSh — APPROVED** [review]

**Review by mxrcury — APPROVED** [review]

**Review by Yurenko — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**